.github/workflows/release.yaml

```diff
@@ -8,6 +8,10 @@ jobs:
     strategy:
       matrix:
         include:
+          - runner: ubicloud-standard-2
+            platform: x64
+            edk2_version: edk2-stable202311
+            edk2_commit: 8736b8fdca85e02933cdb0a13309de14c9799ece
           - runner: ubicloud-standard-2
             platform: x64
             edk2_version: edk2-stable202402
```

Comment:
will the other two artifacts that are not changed in the PR be recreated? if so is it possible that their sha256 changes or is the release process completely deterministic?